js/libs/ui-shared/src/utils/darkMode.ts

```diff
@@ -0,0 +1,19 @@
+const DARK_MODE_CLASS = "pf-v5-theme-dark";
+const mediaQuery = window.matchMedia("(prefers-color-scheme: dark)");
+
+function updateDarkMode(isEnabled: boolean = false) {
```

Comment:
Seems like a value is always passed now, so the default here can be omitted.
```suggestion
function updateDarkMode(isEnabled: boolean) {
```